memgraph_versioned_docs/version-2.4.2/connect-to-memgraph/drivers/php.md

```diff
@@ -39,34 +39,48 @@ following code to it:
 <?php
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Create connection class and specify target host and port
+// Create connection class and specify target host and port, default is localhost
 $conn = new \Bolt\connection\Socket();
 // Create new Bolt instance and provide connection object
 $bolt = new \Bolt\Bolt($conn);
+// Set available bolt versions for Memgraph.
+$this->setProtocolVersions(4.1, 4, 3);
```

Comment:
@stefanak-michal Is this ok? I am not a PHP dev, but should this be:

```php
$bolt->setProtocolVersions(4.1, 4, 3);
```

Where does `$this` comes from?

Reply:
You are correct. Sorry, my mistake.

Reply:
I'll fix it in the new PR propagating changes to some other pages in docs